docs/specifications/tests/Nameserver-TP/nameserver02.md

```diff
@@ -1,61 +1,113 @@
-## NAMESERVER02: Test of EDNS0 support
+# NAMESERVER02: Test of EDNS0 support
```

Comment:
# NAMESERVER02: Test of EDNS0 support, independent of the version

Reply:
The test case does not test ENDS0 is EDNS version 0, and that is what is tested.

Reply:
ok